src/components/support-form/StepperIcon.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import clsx from 'clsx'
+import SettingsIcon from '@material-ui/icons/Settings'
+import GroupAddIcon from '@material-ui/icons/GroupAdd'
+import VideoLabelIcon from '@material-ui/icons/VideoLabel'
+import { StepIconProps } from '@material-ui/core/StepIcon'
+import { AccessibilityNew } from '@material-ui/icons'
+import SecurityIcon from '@material-ui/icons/Security'
+
+const useStepIconStyles = makeStyles({
+  root: {
+    backgroundColor: '#ccc',
+    zIndex: 1,
+    color: '#fff',
+    width: 50,
+    height: 50,
+    display: 'flex',
+    borderRadius: '50%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  active: {
+    backgroundImage: 'linear-gradient( 136deg, #4AC3FF 0%, #29a2df 50%, #1b88be 100%)',
+    boxShadow: '0 4px 10px 0 rgba(0,0,0,.25)',
+  },
+  completed: {
+    backgroundImage: 'linear-gradient( 136deg, #4AC3FF 0%, #29a2df 50%, #1b88be 100%)',
+  },
+})
+
+export default function StepIcon(props: StepIconProps) {
+  const classes = useStepIconStyles()
+  const { active, completed } = props
+
+  const icons: { [index: string]: React.ReactElement } = {
+    1: <SettingsIcon />,
+    2: <GroupAddIcon />,
+    3: <VideoLabelIcon />,
+    4: <SecurityIcon />,
+    5: <AccessibilityNew />,
+  }
```

Comment:
We've got a suggestion by @Svetlina89 to update the icons of the steps as follows:

```
1: `GroupAddIcon`
2: `HelpIcon`
3: `ContactMailIcon`
4: `SecurityIcon` (same)
5: `AccessibilityNew` (same)